### DISCUSSION TITLE
rename the repository?

### DISCUSSION THREAD

**Discussion by dyf** [AllenNeuralDynamics/data_schema#71] [Ideas]
`data_schema` is a somewhat generic name. Especially if we want to make it installable, it will be nice to have something a touch more specific and pythonic.

Proposals:

- `aind-data`
- `aind-data-schema`
- `aind-schema`

@saskiad @jtyoung84 @mekhlakapoor

**Comment by jtyoung84** [comment]
My vote is for `aind-data-schema` to make it consistent with our other projects' naming convention.
